pkg/internal/codegen/parse/index.go

```diff
@@ -208,6 +209,41 @@ func parseResourceTag(tag string) resourceTags {
 	return result
 }
 
+// parseResourceAnnotation parses the tags in a "+resource=" comment into a resourceTags struct.
+func parseResourceAnnotation(t *types.Type) (resourceTags, error) {
+	result := resourceTags{}
+	for _, comment := range t.CommentLines {
+		anno := general.GetAnnotation(comment, "kubebuilder:resource")
+		if len(anno) == 0 {
+			continue
+		}
+		for _, elem := range strings.Split(anno, ",") {
+			key, value, err := general.ParseKV(elem)
+			if err != nil {
+				return resourceTags{}, fmt.Errorf("// +kubebuilder:resource: tags must be key value pairs.  Expected "+
+					"keys [path=<subresourcepath>] "+
+					"Got string: [%s]", comment)
+			}
+			for _, s := range anno {
+				fmt.Printf("\n[debug] %v", s)
+			}
```

Comment:
get rid of these before the final commit.

Reply:
Sure thing!